examples/wms/wms.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: tutorial_v2
+title: Using WMS and TMS services
+---
+
+<style>
+iframe {
+    border: 1px solid #ccc;
+    border-radius: 5px;
+}
+</style>
+
+<br/>
+
+WMS, short for [*web map service*](https://en.wikipedia.org/wiki/Web_Map_Service), is a popular way of publishing maps by professional GIS software (and seldomly used by non-GISers). This format is similar to map tiles, but more generic and not so well optimized for use in web maps. TMS stands for *tiled map service*, and is a WMS-like standard more focused on web maps.
+
+## WMS in Leaflet
+
+When somebody publishes a WMS service, most likely they link to something called a `getCapabilities` document. For this tutorial, we'll use the demo map services from GeoServer, at http://demo.opengeo.org/geoserver/web/. As you can see in that page, "WMS" links to the following URL:
+
+	http://demo.opengeo.org/geoserver/ows?service=wms&version=1.3.0&request=GetCapabilities
+
+Leaflet does not understand WMS GetCapabilities documents. Instead, you have to create a `L.TileLayer.WMS` layer, provide the base WMS URL, and specify whatever WMS options you need.
+
+The base WMS URL is simply the `GetCapabilities` URL, without any parameters, like so:
+
+	http://demo.opengeo.org/geoserver/ows?
+
+And the way to use that in a Leaflet map is simply:
+
+	var map = L.map(mapDiv, mapOptions);
+
+	var wmsLayer = L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', wmsOptions).addTo(map);
+
+An instance of `L.TileLayer.WMS` needs at least one option: `layers`. Be careful, as the concept of "layer" in Leaflet is different from the concept of "layer" in WMS!
+
+WMS servers define a set of *layers* in the service. While these are defined in the `getCapabilities` document, it's simpler to use software such as [QGIS to see what layers are available in a WMS server](http://www.qgistutorials.com/en/docs/working_with_wms.html):
```

Comment:
might mention that a readable XML is returned but that QGIS makes it more readable.